modules/nf-core/minimap2/index/main.nf

```diff
@@ -3,8 +3,8 @@ process MINIMAP2_INDEX {
 
     conda (params.enable_conda ? 'bioconda::minimap2=2.21' : null)
```

Comment:
```suggestion
    // Note: the versions here need to match the versions used in minimap2/align
    conda (params.enable_conda ? 'bioconda::minimap2=2.24' : null)
```

Reply:
And https://github.com/nf-core/modules/blob/42a5fb17f30c047edd307a73f900b0fabb7df3ae/modules/nf-core/minimap2/align/main.nf#L5 should be updated to match the versions of minimap2 and samtools used in the mulled container + a note to keep the versions in sync with minimap2/index